### ISSUE TITLE
Can't create gallery. Strange backend-design

### ISSUE THREAD

**Issue by Fabs87** [valendesigns/option-tree#560]
Hey,

I have a problem using your plugin. I use it in theme-mode and followed your intruction to set it up. All needed files are included and I can create sections and multiple settings.

But when I go to the wp-backend to the design-area and choose the "Theme Options", it looks a litte bit strange. Also I am not able to select images for my gallery.

It seems that, somehow, a stylesheet is missing. Perhaps some other files. I downloaded the plugin from the wordpress-ressources. Please see the attached image.
![view](https://cloud.githubusercontent.com/assets/9106911/9732068/315562f4-5621-11e5-9c25-19afeef22fb7.png)

When I click on "create gallery", nothing happens. I hope you can help me. Your plugin seems to be the one I have been looking for so long :)

**Comment by Fabs87** [comment]
Hi, it's me again. I found the problem causing the strange design. I set the 
```php
add_filter( 'ot_theme_mode', '__return_true' ); 
```

But now, I am nit able to see the different settings which are assigned to one section. In my example, I have assigned 3 settings to the section "Front-Page-Slider". Unfortunatelly it is not possible to see them.
![view2](https://cloud.githubusercontent.com/assets/9106911/9732358/95abc14c-5623-11e5-8e5e-2b767efd9d2e.png)
